scripts/main.js

```diff
@@ -71,4 +71,60 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+console.log(" De afdeling Sale heeft " + departments.sales.numberOfEmployees + "medewerkers");
```

Comment:
Als je met "het laatste stuk" de template literals bedoelt, dan kun je dit doen
```suggestion
console.log(`De afdeling Sale heeft {departments.sales.numberOfEmployees} medewerkers`);
```